ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -50,10 +51,13 @@ StackLayout {
 
     clip: true
 
-    SettingsPageLayout {
+    SettingsPage {
```

Comment:
Why not using here same approach than with minting or permissions, setting `initialItem`?

Reply:
It could be but now it's as it was so far - StackLayout is used as a base.